e2e/specs/add-appointments.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { expect } from '@playwright/test';
+import { generateRandomPatient, deletePatient, type Patient } from '../commands';
+import { test } from '../core';
+import { AppointmentsPage } from '../pages';
+
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('Add appointment for a patient, edit the added appointment and cancel it', async ({ page, api }) => {
+  const appointmentsPage = new AppointmentsPage(page);
+
+  await test.step('When I go to the appointment tab in the patient chart', async () => {
+    await appointmentsPage.goto(patient.uuid);
+  });
+
+  await test.step('And I click on the “Add” button', async () => {
+    await page.getByRole('button', { name: 'Add', exact: true }).click();
+  });
+
+  await test.step('And I select Mobile Clinic location', async () => {
+    await page.getByLabel('Select location').selectOption('Mobile Clinic');
+  });
+
+  await test.step('And I select “Outpatient Department” service', async () => {
+    await page.getByLabel('Select a service').selectOption('Outpatient Department');
+  });
+
+  await test.step('And I make appointment as “Scheduled”', async () => {
+    await page.getByLabel('Select an appointment type').selectOption('Scheduled');
+  });
+
+  await test.step('And I set date for tomorrow', async () => {
+    await page.fill('input[placeholder="dd/mm/yyyy"]', '30/03/2024');
+  });
```

Comment:
```suggestion
  await test.step('And I set date for tomorrow', async () => {
    await page.fill('input[placeholder="dd/mm/yyyy"]', '30/03/2024');
  });
```
Nice to see your progress in tests @Vijaykv5. I would like to highlight that we are using these E2E tests again and again when someone sends a PR. So after the date that u added in this test these tests will fail because this appointment is going to move to the past appointment section. I would like to suggest you if you are adding a date manually make sure add a previous date and in the test make sure to go through the past appointment section.

Reply:
You can use Js to get tomorrow's date. 

ex:
```js
const tomorrow = new Date();
tomorrow.setDate(tomorrow.getDate() + 1);
await page.fill('input[placeholder="dd/mm/yyyy"]', tomorrow.toLocaleDateString('en-GB'));
```

btw, can't we use `page.getByLabel().fill` here?